build.gradle

```diff
@@ -111,6 +112,12 @@ tasks.check.dependsOn integrationTest
 
 gradlePlugin {
     testSourceSets project.sourceSets.integration
+    plugins {
```

Comment:
one last thing - i would rather not change build.gradle with this pr. i entered this comment on sunday or monday, but forgot to "submit the review" :/